src/components/layout/page-layout.tsx

```diff
@@ -0,0 +1,47 @@
+import { Block } from 'baseui/block';
+import React from 'react';
+import { Homepage } from '../../types/homepage';
+import { FooterSection } from '../footer/footer';
+import NavBar from '../nav-bar/nav-bar';
+
+type PageLayoutProps = React.PropsWithChildren<{
+  serverData: any;
+}>;
+
+const PageLayout: React.FC<PageLayoutProps> = ({ children, serverData }) => {
+  const homepage = serverData as Homepage;
+
+  return (
+    <Block minHeight="100vh" display="flex" flexDirection={'column'}>
+      <Block
+        top={0}
+        position={'sticky'}
+        overrides={{
+          Block: {
+            style: () => ({
+              zIndex: 2,
```

Comment:
z-index is a poor way to manage the layers and gets into z-index hell. Please spend time to understand the concept of layers in baseweb - https://baseweb.design/components/layer/ and use it to ensure we are not mutating the z-indexes directly.